### PR TITLE
Busmall charts

### PR DESCRIPTION
**Pull request by sheridp2**

### CODE REVIEW

**Comment by sheridp2** [comment]
Button hover animations added. Chart correctly displays total totals that carry over on refresh of page.

File: app.js [hunk 1]
```diff
@@ -139,6 +139,7 @@ function handleClick(event){
       console.log('Views of ' + stuff[i].name + ' was ' + stuff[i].views + ' and total clicks is ' + stuff[i].clicks);
       console.log('When viewed, the % chance it will be clicked = ' + Math.round((stuff[i].clicks) / (stuff[i].views) * 100));
     }
+    showLink();
```
Comment (fraziermork) [inline app.js]: Nice!

File: app.js [hunk 2]
```diff
@@ -148,3 +149,9 @@ function saveProductsToLocalStorage(stuff){
   localStorage.stuff = JSON.stringify(stuff);
   console.log('saved to localStorage');
 }
+
+function showLink(){
+  var hiddenToNot = document.getElementById('chartsLink');
+  hiddenToNot.removeAttribute('class');
+  hiddenToNot.setAttribute('class', 'chartsButton');
```
Comment (fraziermork) [inline app.js]: Do you think you need to do these two steps separately?

File: charts.js [hunk 3]
```diff
@@ -4,25 +4,51 @@ var stuff = JSON.parse(localStorage.stuff);
...
+
+console.log(localStorage.overallClickTotals);
+// console.log(currentClicks);
+// console.log(localStorage.overallClickTotals);
 // console.log(overallClickTotals);
 // console.log(allStuffClicks(stuff));
 var currentClicks = allStuffClicks(stuff);
 // console.log(currentClicks);
 
+function addToLocalStorage(){
```
Comment (fraziermork) [inline charts.js]: Great!

File: charts.js [hunk 4]
```diff
@@ -4,25 +4,51 @@ var stuff = JSON.parse(localStorage.stuff);
...
+    overallClickTotals.push(cummulClicks);
+    // console.log(overallClickTotals);
+  }
+  localStorage.overallClickTotals = JSON.stringify(overallClickTotals);
+  // console.log(localStorage.overallClickTotals);
+}
+//start of how check to see if click totals are in local storage
 function checkForContent(){
-  if (localStorage.overallClickTotals == 0){
+  if (localStorage.overallClickTotals){
```
Comment (fraziermork) [inline charts.js]: Nice!

File: style.css [hunk 5]
```diff
@@ -2,14 +2,64 @@ body{
...
+  display: inline-block;
+  background-color: yellow;
+  border: 5px solid black;
+  padding: 20px;
+  background: #2db34a;
+  border-radius: 6px;
+  transition: background .2s linear, border-radius 1s ease-in 1s;
+}
+/* The above and below CSS was taken from the Readings 14, http://learn.shayhowe.com/advanced-html-css/transitions-animations/*/
+.returnLink:hover {
```
Comment (fraziermork) [inline style.css]: Ooh

**Pull request merged**